ci/jenkins/test.sh

```diff
@@ -1079,6 +1104,9 @@ elif [[ ${TESTCASE} =~ "e2e" ]]; then
     deliver_antrea
     run_e2e
 else
+    if [[ $TESTBED_TYPE == "kind" ]]; then
```

Comment:
just one small suggestion that instead of adding this inside else you can add this above 1086, instead of using nested statement here.

Reply:
done